packit_service/worker/helpers/testing_farm.py

```diff
@@ -60,40 +61,94 @@ class CommentArguments:
     Parse arguments from trigger comment and provide the attributes to Testing Farm helper.
     """
 
-    packit_command: str = None
-    identifier: str = None
-    labels: List[str] = None
-    pr_argument: str = None
-
     def __init__(self, command_prefix: str, comment: str):
+        self.parser = None
+        self.packit_command: str = None
+        self.identifier: str = None
+        self.labels: List[str] = None
+        self.pr_argument: str = None
+        self.envs: Dict[str, str] = None
+
         if comment is None:
             return
 
-        # Try to parse identifier argument from comment
+        # Init argparse
+        self.parser = self.init_arg_parse
```

Comment:
Shouldn't the method be actually invoked?

```suggestion
        self.parser = self.init_arg_parse()
```

Also, `init_arg_parser()` would make a bit more sense to me.

Reply:
> Shouldn't the method be actually invoked?
That's because of `@property` annotation on the method

> Also, init_arg_parser() would make a bit more sense to me.
I will update it

Reply:
> That's because of `@property` annotation on the method

Oh, sorry, I missed that. But if it's a property then it should be called `arg_parser` or perhaps just `parser`, the verb doesn't belong there.

Reply:
See e.g. here how such a property should be implemented:
https://github.com/packit/packit-service/blob/c376983ead260911b07001f083608f2e5b1246e1/packit_service/worker/handlers/mixin.py#L462-L478

Reply:
Oh I see now, I updated the code to follow this implementation.